src/analytics/src/main/scala/osmesa/analytics/Constants.scala

```diff
@@ -0,0 +1,10 @@
+package osmesa.analytics
+
+object Constants {
+  val ROAD_VALUES =
+    Set(
+      "motorway", "trunk", "motorway_link", "trunk_link", "primary", "secondary", "tertiary",
+      "primary_link", "secondary_link", "tertiary_link", "service", "residential", "unclassified",
+      "living_street", "road"
```

Comment:
Should this include paths, footpaths, cycleways, bridleways, etc. or is this intended for "roads" only vs. navigable features?

Reply:
We're sticking with roads for now, but we should refactor to work with arbitrary topics at a later point.